src/helpers/notificationHelper.js

```diff
@@ -0,0 +1,59 @@
+import socketIOCient from 'socket.io-client';
+import dotenv from 'dotenv';
+import { toast } from 'react-toastify';
+import store from '../redux/store';
+import { NEW_NOTIFICATION } from '../redux/actionTypes/notificationActionTypes';
+
+dotenv.config();
+
+const initializeSocket = () => {
+  const BASE_URL = 'http://localhost:3000/';
+  const HEADERS_REQUEST = {
+    token: localStorage.getItem('token'),
```

Comment:
Why our BASE_URL is calling localhost?

Reply:
I have changed once my backed PR was merged